openwisp_monitoring/tests/test_selenium.py

```diff
@@ -153,3 +156,63 @@ def test_restoring_deleted_device(self):
             AlertSettings.objects.filter(id__in=device_alert_setting_ids).count(), 2
         )
         self.assertEqual(Chart.objects.filter(id__in=device_chart_ids).count(), 3)
+
+
+class TestDashboardCharts(
+    SeleniumTestMixin, TestDeviceMonitoringMixin, StaticLiveServerTestCase
+):
+    def test_dashboard_timeseries_charts(self):
+        self.login()
+        try:
+            WebDriverWait(self.web_driver, 5).until(
+                EC.visibility_of_element_located(
+                    (By.XPATH, '//*[@id="ow-chart-inner-container"]')
```

Comment:
can we use CSS instead of xpath when possible please? It's more familiar to most people.